packages/breadcrumb/src/index.ts

```diff
@@ -1 +1,7 @@
-export { default } from './react/index'
+import '@pluralsight/ps-design-system-button/styles.css'
```

Comment:
So  if you run storybook on this it will error. I ran into the same thing on Select. If you import a component styles that also imports another component styles it throws the error below in storybook. 

<img width="1678" alt="Screen Shot 2021-08-27 at 11 13 30 PM" src="https://user-images.githubusercontent.com/1058725/131205806-6fe7b308-2382-49a3-9bbf-1eb6e576ccbf.png">